.github/workflows/rust.yml

```diff
@@ -68,5 +68,52 @@ jobs:
       env:
         RUSTFLAGS: --cfg tokio_unstable
 
+    - name: Test
+      run: cargo test --verbose --workspace --all-features
+
+  test_linux-non-amd64:
+    name: rust (linux/${{ matrix.arch }})
+    needs: test_amd64
```

Comment:
I am assuming you are adding this `needs` for debugging purposes only? In the end we want all tests to run concurrently (for sure).